ipatests/test_integration/test_krbtpolicy.py

```diff
@@ -238,6 +238,30 @@ def test_krbtpolicy_jitter_otp(self, reset_to_default_policy):
                                  "--user-auth-type", "otp"])
         kinit_check_life(self.master, USER1)
 
+    def test_krbtpolicy_jitter_otp_maxlife(self, reset_to_default_policy):
+        """Test jitter lifetime with OTP"""
+        master = self.master
+        reset_to_default_policy(self.master, USER1)
+        otpuid, hotp = add_otptoken(master, USER1, otptype="hotp")
+        master.run_command(["ipa", "otptoken-show", otpuid])
+
+        master.run_command(['ipa', 'krbtpolicy-mod', USER1,
+                            '--otp-maxlife=172800'])
+        master.run_command(["ipa", "user-mod", USER1,
+                            "--user-auth-type", "otp"])
+        otpvalue = hotp.generate(0).decode("ascii")
+        # basically kinit_otp
+        tasks.kdestroy_all(master)
+        master.run_command(["kinit", "-n", "-c", "/tmp/armor"])
+        master.run_command(
+            ["kinit", "-T", "-l", "2d", "/tmp/armor", USER1],
```

Comment:
The command should have `"/tmp/armor"` right after `"-T"`.
This is what is seen in the test execution otherwise:
```
INFO     ipatests.pytest_ipa.integration.host.Host.master.IPAOpenSSHTransport:transport.py:391 RUN ['kinit', '-T', '-l', '2d', '/tmp/armor', 'testuser1']
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:513 RUN ['kinit', '-T', '-l', '2d', '/tmp/armor', 'testuser1']
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 Extra arguments (starting with "/tmp/armor").
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 Usage: kinit [-V] [-l lifetime] [-s start_time] [-r renewable_life]
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	[-f | -F] [-p | -P] [-n] [-a | -A] [-C] [-E]
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	[--request-pac | --no-request-pac] 
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	[-v] [-R] [-k [-i|-t keytab_file]] [-c cachename]
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	[-S service_name] [-I input_ccache] [-T ticket_armor_cache]
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	[-X <attribute>[=<value>]] [principal]
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557     options:
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-V verbose
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-l lifetime
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-s start time
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-r renewable lifetime
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-f forwardable
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-F not forwardable
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-p proxiable
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-P not proxiable
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-n anonymous
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-a include addresses
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-A do not include addresses
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-v validate
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-R renew
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-C canonicalize
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-E client is enterprise principal name
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-k use keytab
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-i use default client keytab (with -k)
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-t filename of keytab to use
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-c Kerberos 5 cache name
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-S service
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-I input credential cache
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-T armor credential cache
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	-X <attribute>[=<value>]
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:557 	--{,no}-request-pac request KDC include/exclude a PAC
DEBUG    ipatests.pytest_ipa.integration.host.Host.master.cmd189:transport.py:217 Exit code: 2
```
Please fix it.

Reply:
D'oh. That's what I get for pushing out a change too quickly :-(. Fixed.